pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -23,6 +25,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/snmp/internal/valuestore"
 )
 
+const interfaceStatusMetric = "snmp.interface_status"
```

Comment:
```suggestion
const interfaceStatusMetric = "snmp.interface.status"
```

Should we use this ☝️ to be consistent with:

```

	deviceReachableMetric   = "snmp.device.reachable"
	deviceUnreachableMetric = "snmp.device.unreachable"
```

https://github.com/DataDog/datadog-agent/pull/14649/files#diff-7842570fd2a86f2c84d11bf86caf3a8b367b393ac8868369ad7f8dc822cd3346R37-R38

Reply:
`snmp.interface.XX` make it seems like we could have more metrics under `snmp.interface` than the status, that's why I initially went this direction. For instance things like the interface bandwidth etc should maybe be in this category but they are not.

No strong opinion, the argument for being consistent with snmp.device.reachable is a good point as well

Reply:
I think if in the future, if we have more custom metrics related to interfaces we can use the same hierarchy `snmp.interface.new_metric`.

> For instance things like the interface bandwidth etc should maybe be in this category but they are not.

Yes, indeed. However, I think metrics derived from MIB symbols is a different case, we probably have no choice to continue the naming convention of `snmp.myMIBSymbolName`.